### DISCUSSION TITLE
Charter discussion

### DISCUSSION THREAD

**Discussion by lmbr-pip** [o3de/sig-operations#23] [General]
As per the last sig-operations meeting, started discussion about charter updates. Please add comments, questions, thoughts to this discussion so we can quickly bring the charter up-to-date.

Here is a partial update of the charter:

# Operations SIG Charter

## Overview of SIG

The Operations SIG ("the SIG") maintains the operation and integration of tools to run Github & Discord automations and other related community tools. 

## Goals

The goal of sig-operations is to own and create automations that remove monotony of repeatable tasks around communication tasks for O3DE. The SIG does not own build or engine code.

### Community goals

TBD

## Scope

* Discord
* Discord Bots
* GitHub actions including issue manipulation
* Automations not tied to code/builds

## In scope

* Discord
    * Tools, emojis hooks 
    * Discord Bots
* GitHub hooks and actions.
  * DCO setup
* Automation review and management of GitHub repros

## Cross-cutting Processes

* Tool instance management for automation with #sig-build 
* Code running on website for search integration, javascript widgets with #sig-docs-community and #sig-build
* robot.txt management (related to tooling) with #sig-build

## Out of Scope
* Management of O3DE YouTube channel. Channel is owned by an "infrastructure group". Content is responsibility in sig-docs-community
* Management of the O3DE website
* Any code relating to build automation`

## SIG Links and lists

* Joining
* Slack/Discord: https://docs.o3de.org/community/ 
* YouTube Channel: https://www.youtube.com/c/Open3DEngine
* Forums: https://github.com/o3de/sig-operations/discussions
* Mailing list: https://lists.o3de.org/g/sig-operations
* Issues/PRs: https://github.com/o3de/sig-operations/issues, https://github.com/o3de/sig-operations/pulls
* Meeting agenda & Notes: https://github.com/o3de/sig-operations/tree/main/meetings/notes 

## Roles and Organization Management

The Documentation and Community SIG adheres to the standards for roles and organization management as specified by O3DE charter. This SIG opts in to updates and modifications to.

## Individual Contributors

If you are targeting a particular release, make sure the pull request and issue is marked with the corresponding release milestone. This must be the same release identifier for both docs and code commits.

## Maintainers

Each release cycle, the current SIG Chair(s) must update membership. Maintainers of the GitHub team are defined as follows:

* SIG chair(s): TBD
* Documentation lead / sub-chair
* Community lead / sub-chair

## Additional responsibilities of Chairs

Chairs also serve as Tech Leads. Chairs are responsible for coordinating efforts between the sub-SIGs as necessary.

## Subproject Creation

SIG Chairs can create subprojects without requiring member votes. Subprojects may be found in the subproject directory.

## Deviations from sig-governance

Per readme:

* Meetings are weekly, every thursday
* Once per quarter, leads and other interested parties meet to discuss quarterly goals and achievements

**Comment by FiniteStateGit** [comment]
Is long-term management/distribution of the credentials to external sites in-scope?

**Reply by lmbr-pip** [reply]
May be worth iterating which credentials we think are 'owned' / cross-cutting for the SIG
